packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -163,7 +165,7 @@ def main():
             log.error('Unknown service: {}'.format(service))
             sys.exit(1)
         apply_service_configuration(service)
-        log.debug('bootstrapping {}'.format(service))
+        log.info('bootstrapping {}'.format(service))
```

Comment:
❓ would it be useful to add `service` name into logs output as well?

Reply:
I'm conscious that we're trying to keep log size down. The combination of the pid along with this line, associating the pid with the service name should be enough.